models/production/classroom_surveys/schema.yml

```diff
@@ -94,14 +109,20 @@ models:
         description: "Submission date"
         tests:
           - not_null
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: bigint
```

Comment:
how is this working? this should be date field right?

Reply:
yes, you're right! let me fix this